src/test/groovy/transform/stc/DefaultGroovyMethodsSTCTest.groovy

```diff
@@ -37,6 +37,20 @@ class DefaultGroovyMethodsSTCTest extends StaticTypeCheckingTestCase {
                 println it // DGM#println(Object,Object)
             }
         '''
+
+        assertScript '''
+            int[] nums = [0, 1]
+            nums.each { it -> // ArrayGroovyMethods#each(int[], Closure)
+                println it.toDouble() // DGM#println(Object,Object)
```

Comment:
Though we have many legacy tests which use println, we tend to avoid adding additional such tests when we can since they slow down the build and don't actually get automatically tested (even though the main objective here is for compilation to pass).

Reply:
I think I would stop writing tests in this test class and only use the tests inside the Javadoc since otherwise the number of tests would explode for ~60 methods with each 8 primitive types (roundabout 480 tests).